taskcluster/scripts/pipeline/train_taskcluster.py

```diff
@@ -26,7 +26,6 @@
     "opustrainer.log",
     "train.log",
     "valid.log",
-    "vocab.spm",
```

Comment:
I believe this is the code that can continue from an arbitrary training run. In your description of this PR you mention we'll need to rename the GCS artifacts. I would prefer to not break training continuation, as we'll continue adding artifacts to the GCS bucket in a somewhat arbitrary fashion until it is fully automated, and I'm about to work with continuation. We can't migrate everything in one pass with this chnage.

I would prefer to handle the two cases here for historic runs with some additional logic in:

```
for artifact in resp.json()["artifacts"]:
```

You could make it aware of the single vocab case, and then handle copying the artifacts to the proper place.